### DISCUSSION TITLE
[Feature Request] Show query of panels which are readonly

### DISCUSSION THREAD

**Discussion by hikhvar** [grafana/grafana#60036] [Dashboards, Panels & Visualizations]
# What would you like to be added

I would like to easily see the queries plotted in Grafana panels sourced via the dynamic provisioning. Currently, I have to go into the inspect mode to have a look at the dashboard json to find the query executed by a panel. 
I need this only for read-only dashboards, where I have the editor capabilities. I think this is not required for public dashboards. 

# Why is this needed (describe your use case and goals)

Especially in the Kubernetes ecosystem, there are a lot of tools which come with dashboards included. While many are of good quality, there are also a lot of panels where it is very hard to figure out what is plotted at all. Either because there are no units on the axis, the legend is miss-leading, there is no description of the panel, or just a generic title. So to interpret the data, it would be very cool to see the underlying query issued to the datasource.

Another use-case, is to redefine the query ad-hoc in the Explore function.

**Comment by joshhunt** [comment]
The `viewers_can_edit` server config option should allow this https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#viewers_can_edit

> Viewers can access and use [Explore](https://grafana.com/docs/grafana/latest/explore/) and perform temporary edits on panels in dashboards they have access to. They cannot save their changes. Default is false.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
